electrum/gui/qml/components/controls/HistoryItemDelegate.qml

```diff
@@ -108,7 +108,11 @@ Item {
                         } else if (Daemon.fx.historicRates) {
```

Comment:
```suggestion
                        } else if (Daemon.fx.historicRates && model.timestamp) {
```
(pre-existing bug regardless of PR: if historyRates is enabled, for unconfirmed (or local) txs, the fiat prices is not shown)